automated_analysis.py

```diff
@@ -382,50 +382,54 @@ def set_survey_percentages(survey_counts, total_survey_counts):
     plt.savefig(f"{automated_analysis_output_dir}/maps/regions/regions_total_participants.png", dpi=1200, bbox_inches="tight")
     plt.close()
 
-    for plan in PipelineConfiguration.RQA_CODING_PLANS:
-        episode = episodes[plan.raw_field]
-
-        for cc in plan.coding_configurations:
-            # Plot a map of the total relevant participants for this coding configuration.
-            rqa_total_region_frequencies = dict()
-            for region_code in CodeSchemes.SOMALIA_REGION.codes:
-                if region_code.code_type == CodeTypes.NORMAL:
-                    rqa_total_region_frequencies[region_code.string_value] = \
-                        episode["Total Relevant Participants"][f"region:{region_code.string_value}"]
-
-            fig, ax = plt.subplots()
-            MappingUtils.plot_frequency_map(regions_map, "ADM1_AVF", rqa_total_region_frequencies,
-                                            label_position_columns=("ADM1_LX", "ADM1_LY"),
-                                            callout_position_columns=("ADM1_CALLX", "ADM1_CALLY"), ax=ax)
-            plt.savefig(f"{automated_analysis_output_dir}/maps/regions/region_{cc.analysis_file_key}_1_total_relevant.png",
-                        dpi=1200, bbox_inches="tight")
-            plt.close()
-
-            # Plot maps of each of the normal themes for this coding configuration.
-            map_index = 2  # (index 1 was used in the total relevant map's filename).
-            for code in cc.code_scheme.codes:
-                if code.code_type != CodeTypes.NORMAL:
-                    continue
-
-                theme = f"{cc.analysis_file_key}_{code.string_value}"
-                log.info(f"Generating a map of per-region participation for {theme}...")
-                demographic_counts = episode[theme]
+    generate_region_theme_distribution_maps = False #TODO: move this to config file?
```

Comment:
If this is optional, you'll want to make the upload optional too, or clear the automated analysis folder between runs

Reply:
will update this in this pr https://github.com/AfricasVoices/Project-IMAQAL-COVID19/pull/50 now that we have automated analysis configs in configuration files. Thanks